client/src/views/TableDetail.vue

```diff
@@ -2,27 +2,23 @@
 <div>
 <h1>Table: {{`${this.workspace}/${this.table}`}}</h1>
   <table>
-  <thead>
-    <tr >
-    <th v-for="head in this.headers" :key="head" class="head">
-      {{head}}
-    </th>
-    </tr>
-  </thead>
-  <tbody v-for="(row, index) in rowKeys" :key="row[0].value" class="row-wrap">
-
-    <tr :class="classNameByIndex(index)">
-      <td v-for="col in row" :key="col.key" class="col">
-      {{col.value}}
-    </td>
-    </tr>
-
-  </tbody>
-
+    <thead>
+      <tr >
+      <th v-for="head in this.headers" :key="head" class="head">
+        {{head}}
+      </th>
+      </tr>
+    </thead>
+    <tbody v-for="(row, index) in rowKeys" :key="row[0].value" class="row-wrap">
+      <tr :class="rowClassName(index)">
+        <td v-for="col in row" :key="col.key" class="col">
+        {{col.value}}
```

Comment:
Can you indent lines 15 and 16 with two more spaces?

Reply:
fixed those!